src/sentry/api/urls.py

```diff
@@ -596,6 +596,11 @@
         OrganizationEventsEndpoint.as_view(),
         name='sentry-api-0-organization-events'
     ),
+    url(
+        r'^organizations/(?P<organization_slug>[^\/]+)/events/(?P<event_id>[^\/]+)/json/$',
```

Comment:
I think we *might* need a project id here, as eventids in snuba are only guaranteed unique within a project.

Reply:
@markstory I totally agree. I'll add that to the url. That will simplify my code I think.